app/pages/admin/project-status/toggle.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import AutoSave from '../../../components/auto-save';
+
+class Toggle extends Component {
+  constructor(props) {
+    super(props);
+    this.set = this.set.bind(this);
+    this.state = {
+      error: null,
+      setting: { }
+    };
+  }
+
+  set(property, value) {
```

Comment:
This was a pretty literal translation of the mixin code.  Should be able to improve it